### DISCUSSION TITLE
Burn - dead round (martingale issue)

### DISCUSSION THREAD

**Discussion by ghost** [lmusarella/Pancakeswap-Prediction-Bot-2023#20] [Ideas]
The bot doesn't see a dead round, where it ends up as the same value as the previous (pancakeswap calls it a 'burn' round). 

This is an issue because the bot doesn't continue with the martingale as it sees it as a win, and you lose all BNB as it restarts the martingale.

This needs to be fixed.

Thanks for your help.

**Comment by lmusarella** [comment]
Hi, in theory I already implemented a fix that should handle "burnt" rounds. Did you get this problem again with the current version?

**Comment by lmusarella** [comment]
Fixed!
